frontend/lib/item_listing/components/filter_bottom_sheet/filter_bottom_sheet.component.dart

```diff
@@ -0,0 +1,427 @@
+              const SizedBox(height: 16),
+              _buildPriceRangeButton(context, filterViewModel),
+              const SizedBox(height: 16),
+              _buildDateSection(context, filterViewModel),
+              const SizedBox(height: 16),
+              _buildConditionSection(filterViewModel),
+              const SizedBox(height: 24),
+              _buildButtons(context, filterViewModel),
+            ],
+          ),
+        );
+      },
+    );
+  }
+
+  Widget _buildCampusSection(FilterViewModel viewModel) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        const Text(
+          'Campus',
+          style: TextStyle(
+            fontSize: 20,
+            fontWeight: FontWeight.w500,
+          ),
+        ),
+        const SizedBox(height: 12),
+        Wrap(
+          spacing: 8,
+          runSpacing: 8,
+          children: ['Mississauga', 'St. George', 'Scarborough'].map((campus) {
+            final isSelected = viewModel.campus == campus;
+            return FilterChip(
+              padding: const EdgeInsets.symmetric(horizontal: 8),
+              materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
+              label: Text(
+                campus,
+                style: TextStyle(
+                  fontSize: 13.1,
+                  color: isSelected ? const Color(0xFF2F9CCF) : Colors.black,
+                  fontWeight: isSelected ? FontWeight.w500 : FontWeight.normal,
+                ),
+              ),
+              selected: isSelected,
+              onSelected: (selected) {
+                viewModel.setCampus(selected ? campus : null);
+              },
+              backgroundColor: Colors.grey[200],
+              selectedColor: const Color(0xFF2F9CCF).withAlpha(51),
+              checkmarkColor: const Color(0xFF2F9CCF),
+            );
+          }).toList(),
+        ),
+      ],
+    );
+  }
+
+  Widget _buildSortingSection(FilterViewModel viewModel) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        const Text(
+          'Sort By',
+          style: TextStyle(
+            fontSize: 20,
+            fontWeight: FontWeight.w500,
+          ),
+        ),
+        const SizedBox(height: 12),
+        Container(
+          decoration: BoxDecoration(
+            color: Colors.grey[200],
+            borderRadius: BorderRadius.circular(12),
+          ),
+          child: Column(
+            children: [
+              _buildSortOption(
+                'Price: Lowest first',
+                SortOrder.priceLowToHigh,
+                viewModel,
+              ),
+              Divider(height: 1, color: Colors.grey[300]),
+              _buildSortOption(
+                'Price: Highest first',
+                SortOrder.priceHighToLow,
+                viewModel,
+              ),
+            ],
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _buildSortOption(
```

Comment:
Are we supporting sorting by date? 
<img width="426" alt="image" src="https://github.com/user-attachments/assets/6633fd8d-2e0c-4737-9641-74e001eaf0eb" />